js/components/MagnetMapView.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component, PropTypes } from 'react';
+import {
+  View,
+  Dimensions,
+  StyleSheet,
+} from 'react-native';
+import MapView from 'react-native-maps';
+
+export default class MagnetMapView extends Component {
+  render() {
+    const { height, width } = Dimensions.get('window');
```

Comment:
This is easy to do through props and styling.
At this stage I'd like to avoid polishing components and trying to suit all possible use cases while we don't have UX mockups or UI design. Let's avoid early optimisations of components.